cameo/strain_design/core.py

```diff
@@ -0,0 +1,319 @@
+# Copyright 2015 Novo Nordisk Foundation Center for Biosustainability, DTU.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from functools import partial
+
+import sys
+from cobra import DictList
+from pandas import DataFrame
+
+from cameo import ui
+from cameo.core.result import Result
+
+try:
+    from gnomic import Genotype
+    from gnomic.models import Feature, Accession, Del, Ins, Sub, Mutation
+    _gnomic_available = True
+except Exception:
```

Comment:
I mean, it doesn't raise an ImportError, it raises a SyntaxError.
Here:
https://travis-ci.org/biosustain/cameo/jobs/174900373#L1072